### PR TITLE
FAQ Page UI Enhancement

### PR DESCRIPTION
**Pull request by amanprateek123**
## Issue that this pull request solves
Enhancing the UI of FAQ page.
 Closes: # (issue number)

## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [x] Other (please describe): UI issue

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x ] My changes generate no new warnings
- [ x] My changes does not break the current system and it passes all the current test cases.

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface

## Other information

Any other information that is important to this pull request

### CODE REVIEW

**Comment by mohit355** [comment]
@amanprateek123 
1.  link the issue that this PR solving.
2. Describe the changes done by you.
3.  Attach snapshots of enhanced UI.

**Comment by amanprateek123** [comment]
@mohit355  There is no issue number regarding this PR. @Kajol-Kumari  tell me to just send the PR . I changed the UI of FAQ page.
Screenshot from 2021-01-05 13-45-50

**Comment by mohit355** [comment]
Attach the issue number as a link

**Review by mohit355 — COMMENTED** [review]
proper indentation required

File: frontend/src/components/faq/faqs.css [hunk 1]
```diff
@@ -1,86 +1,60 @@
...
+  width: 80%;
+  margin: 0em auto 1em auto;
+  display: flex;
+  height: auto;
+  align-items: center;
+}
+.faq-image{
+  width: 42%;
+}
+.faq-block{
```
Comment (Kajol-Kumari) [inline frontend/src/components/faq/faqs.css:18]: please leave one line space here

File: frontend/src/components/faq/faqs.css [hunk 2]
```diff
@@ -1,86 +1,60 @@
...
-.faq-block {
-  box-shadow: 0 4px 2px 0 rgba(0, 0, 0, 0.19);
+.faq{
+  width: 80%;
+  margin: 0em auto 1em auto;
+  display: flex;
+  height: auto;
+  align-items: center;
+}
+.faq-image{
```
Comment (Kajol-Kumari) [inline frontend/src/components/faq/faqs.css:15]: please leave one line space at the end of each css class

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
Hey @amanprateek123 design looks preety good just format the code properly using `preetier` and it will be good to go :)

**Comment by amanprateek123** [comment]
Please open the issue and give me the issue number for this. And for rest I will make the changes as mentioned and will send the PR again.

**Pull request closed without merging**